tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -552,12 +552,11 @@ EOT
 			LIBC=gnu
 			echo "WARNING: Forcing LIBC=gnu because $ARCH has no musl Rust target"
 		fi
-		[ "$LIBC" == "musl" ] && bash ${script_dir}/../../../ci/install_musl.sh
 		# rust agent needs ${arch}-unknown-linux-${LIBC}
+		test -r "${HOME}/.cargo/env" && source "${HOME}/.cargo/env"
 		if ! (rustup show | grep -v linux-${LIBC} > /dev/null); then
-			bash ${script_dir}/../../../ci/install_rust.sh ${RUST_VERSION}
```

Comment:
This change will break some CI jobs.

I did grep the tests repository, it turned out that `tests/.ci/jenkins_job_build.sh` and `tests/tracing/test-agent-shutdown.sh` are the only scripts calling the `install_rust.sh` script. The former is used by most of jobs on Jenkins ( that's why some jobs ran fine with that change) but others won't use it; examples are  `tests/.ci/openshift-ci/build_install.sh` (used on OpenShift CI) and `tests/.ci/vfio_jenkins_job_build.sh` (entry point of the VFIO job). Also if I run a CI job locally with `.ci/setup.sh && .ci/run.sh` it will fail too (I guess).

I don't like that Kata scripts install stuffs on developer's system so I find this change welcomed. We just need to find the best place to call the `install_rust.sh` script to disturb the fewest jobs (use cases) possible.

Reply:
Right, I noticed the problem, I just haven't had a chance to debug yet (hence `wip`).  I'm thinking I'll probably take a different approach to this whole thing.